lib/blemulator.dart

```diff
@@ -1,3 +1,7 @@
+/// Library with all BLEmulator definitions
```

Comment:
Does the amount of slash make a difference? I noticed that in some places comment starts with double slash and in the others it starts with triple slash.

Reply:
`///` are C#-style dart docs